setup.py

```diff
@@ -173,7 +173,7 @@ def run(self):
         'jupyter_bokeh >=3.0.7',
         'django <4',
         'channels',
-        'pyvista <0.33',
+        'pyvista',
```

Comment:
Readd pin to determine if it causes the tests to stall:

```suggestion
        'pyvista <0.33',
```